### ISSUE TITLE
A11y_WINFORMS_Remaining_Controls_Testing_Remaining-Controls_Zoom:At 200% zoom item and value section is not visible clearly

### ISSUE THREAD

**Issue by Durgajyothi143** [dotnet/winforms#7355]
### .NET version

.NET Framework Version 4.8.04161
Operating System: Windows 11 Enterprise 21H2

### Did it work in .NET Framework?

No

### Issue description

GitHubTags:#A11yMAS;#A11yTCS;#.NET Core[WINFORMS]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.4;#DesktopApp;#Win11;#Benchmark;#Zoom

**Actual:**

At 200% zoom item and value scetion is  not visible clearly

**Expected:**

At 200% zoom item and value section should be visible clearly

**User Impact:**

User with low vision will get difficulty to see the content

### Steps to reproduce

1. Open Core testing solution in VS 2019 Int Preview
2. Right click on cs.proj file and click on Build.
3.Then run the application. 
4.Main Form screen would appear.
5.Navigate to Remaining_Controls_Testing Click on this.
6.Remaining Controls screen would appear.

**Attachment:**
![A11y_WINFORMS_Remaining_Controls_Testing_Remaining-Controls_Zoom #7355](https://user-images.githubusercontent.com/108055767/176091705-0e7903c2-84dd-4fe0-9058-36bd2cd06f30.png)

**Comment by merriemcgaw** [comment]
Note there is a scrollbar on the form and that allows the full control to be accessible without "loss of content or functionality".

**Comment by merriemcgaw** [comment]
@Olina-Zhang can you get a full screen shot?

**Comment by Durgajyothi143** [comment]
Hi @merriemcgaw 
I have used Display Resolution :1366*768 before moving to 200% custom scaling .

**Comment by Olina-Zhang** [comment]
@merriemcgaw I verified this form page on display resolution: 1366*768, set Magnifier with 200% custom scaling. In this mode, we can move mouse to see all form easily, please see following gif:
![200Zoom](https://user-images.githubusercontent.com/26474449/176404697-665a8737-00f9-4654-86b4-1bef593abf4f.gif)

**Comment by merriemcgaw** [comment]
@Olina-Zhang it looks like this dialog has a horizontal scrollbar at the bottom of the form, as well as the vertical one on the side. Can you please take a look and remove that horizontal scrollbar? That's the part that we are not allowed to have. The vertical one is fine, the horizontal one is not. And I see it at the 100% scaling, so I think it may have been added to demonstrate that forms can scroll, but we should just remove it for testing purposes.

**Comment by Olina-Zhang** [comment]
@merriemcgaw  I have redesigned this form page to make the controls a little more compact to shrink the form without scrollbar displayed. And tested it at 200% scaling with the resolution at 1024x768. Can move mouse to see all screens easily:
![200ZoomCommon1](https://user-images.githubusercontent.com/26474449/176654656-da93f880-564a-41a3-9891-50e0e9dd22a9.gif)

**Comment by merriemcgaw** [comment]
@Olina-Zhang that looks great. Are you able to see the whole form at once when you've set the dpi scaling to 200% not just the magnifier (though magnifier can be super helpful lots of times). I think the rules they're looking for are 1280x768 at 200% dpi scaling.

**Comment by merriemcgaw** [comment]
1280x1024 with a dpi scaling of 200% shows the following
<img width="437" alt="OK at 200 percent" src="https://user-images.githubusercontent.com/20996271/176794916-45270672-fa4a-4d8f-9afb-98255b6e4677.png">
:

**Comment by merriemcgaw** [comment]
@Durgajyothi143 your screen size is too small. For desktop apps we can test at 1280x10024 and 200% dpi scaling. The above snapshots is what I see when I do so. Please revalidate all scaling related issues at the new screen size.

**Comment by Durgajyothi143** [comment]
Verified the bug as per above comment issue is not reproducing. Hence closing the bug
![Screenshot (43)](https://user-images.githubusercontent.com/108055767/179243497-675b730a-a9a0-4c12-a9ad-14ce28879f57.png)

**Comment by Tanya-Solyanik** [comment]
Closing per request from Samhitha K S (TATA CONSULTANCY SERVICES LTD)

Issue is fixed. Adding #Closed; tag.
